py_zipkin/zipkin.py

```diff
@@ -494,10 +493,18 @@ def stop(self, _exc_type=None, _exc_value=None, _exc_traceback=None):
         # process (i.e. this zipkin_span not inside of any other local
         # zipkin_spans)
         if self.logging_context:
-            self.logging_context.stop()
-            self.logging_context = None
-            self.get_tracer().set_transport_configured(configured=False)
-            return
+            try:
+                self.logging_context.stop()
+            except Exception:
+                self.get_tracer().clear()
+                err_msg = 'Error while emitting zipkin trace. {}'.format(
+                    self.service_name,
+                )
+                log.error(err_msg)
```

Comment:
This is not a very useful message as it doesn't tell you what went wrong. Can we at least print the exception `repr`?